docs/synapse/userguides/ug015_storm_ref_pivot.rst

```diff
@@ -11,85 +15,339 @@ Where specific query examples are given, they are commonly provided in pairs usi
 
 pivot()
 -------
-Todo
+Returns a set of nodes that share a specified property of the same type / valu as the original set of nodes.
+
+Optional parameters:
+
+* **Return limit:** specify the maximum number of nodes returned by the ``pivot()`` query.
+
+  * ``limit=`` (operational syntax)
+  * ``^`` (macro syntax)
+
+**Operator syntax:**
+
+.. parsed-literal::
+
+  **pivot(** [ *<srcprop>* **,** ] *<dstprop>* [ **, limit=** *<limit>* ] **)**
+
+**Macro syntax:**
+
+.. parsed-literal::
+
+  [ *<srcprop>* **^** *<limit>* ] **->** *<dstprop>*
```

Comment:
So just to clarify, using `limit=` in `pivot(...)` will limit the number of nodes lifted which we would pivot TO from the source, but in the macro example, the `limit` is limiting the number of nodes we would initially lift (and thus pivot from)

Reply:
Actually it seems to operate the same either way - it seems to be an anomaly based on the generic macro syntax (which has to account for using "limit") whether it's a lift, filter, pivot, etc.

Reply:
(but checking)

Reply:
You're correct. Removed all references to limit ( ^ ) in Macro syntax, fixed Usage statement, example using "limit", and Usage notes.